.github/workflows/build_main_documentation.yml

```diff
@@ -95,15 +99,29 @@ jobs:
           cp doc-build/${{ inputs.package }}/_versions.yml build_dir/${{ inputs.package }}
 
       - name: Make documentation
+        shell: bash
         env:
           NODE_OPTIONS: --max-old-space-size=6656
         run: |
           echo "doc_folder has been set to ${{ env.doc_folder }}"
 
-          if [ -z "${{ inputs.notebook_folder }}" ]
+          if [ -z "${{ inputs.notebook_folder }}" ];
```

Comment:
We're reaching the point where this should be tested on the side and add something to an `EXTRA_ARGS` command, cause all your stuff about language will also need to be in the lese branch of this test ;-)

Reply:
I've now grouped all the base arguments to `doc-builder` in an `args` command here: https://github.com/huggingface/doc-builder/pull/164/commits/dd5d190b2545d76d582a01defcfb63d2036a5faf

I then update `args` if the user provides the `notebook_folder` input and the result is now much simpler :)

The test builds for the [`course`](https://github.com/huggingface/course/runs/5743887075?check_suite_focus=true) and [`datasets` ](https://github.com/huggingface/datasets/runs/5743888058?check_suite_focus=true) PRs still pass, so the last check is to make sure a build of the main docs also works.

Reply:
Here's a test run that we can still build the `datasets` docs with `build_main_documentation.yml` (looks good): https://github.com/huggingface/datasets/runs/5744097672?check_suite_focus=true